client/extensions/woocommerce/app/settings/payments/index.js

```diff
@@ -92,6 +91,45 @@ class SettingsPayments extends Component {
 		this.props.createPaymentSettingsActionList( successAction, failureAction );
 	};
 
+	renderPaymentSection = paymentSection => {
+		return (
```

Comment:
Change the `=> { ` to ` `=> (` and you don't need the `return`

Reply:
Good idea - will do!

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/20928/commits/71969111fdc801cecfb52fd07ec0c7b9086a1afe